test/integrators/autodiff_events.jl

```diff
@@ -22,3 +33,24 @@ ad = ForwardDiff.jacobian(test_f,p)
 ad
 
 @test ad ≈ findiff
+
+function test_f2(p, sensealg=ForwardDiffSensitivity(), controller=nothing)
+  _prob = remake(prob, p=p)
+  u = Array(solve(_prob,Tsit5(),sensealg=sensealg,controller=controller,
```

Comment:
Why do you need to call `Array` and just to get the last element?

Reply:
you don't.

Reply:
I changed it to `u[end][end]`.